### PR TITLE
Deploy to stage k8s cluster

### PR DESCRIPTION
**Pull request by ypoplavs**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [x] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by vsukhin — APPROVED** [review]

**Review by nicufk — APPROVED** [review]

**Review by jasmingacic — APPROVED** [review]

**Review by exu — APPROVED** [review]

File: .github/workflows/helm-deploy-testkube-charts-stage.yaml [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+      - name: Installing repositories
+        run: |
+          helm repo add helm-charts https://kubeshop.github.io/helm-charts
+          helm repo add bitnami https://charts.bitnami.com/bitnami
+
+      # Deploy the Docker image to the GKE cluster
+      - name: Deploy
+        run: |-
+          helm upgrade --install --atomic --timeout 180s testkube helm-charts/testkube --namespace testkube --create-namespace --set testkube-api.image.tag=${{ github.event.client_payload.image_tag }} --values ./charts/testkube/values-demo.yaml
```
Comment (exu) [inline .github/workflows/helm-deploy-testkube-charts-stage.yaml:62]: Values for staging are needed + ingress DNS host

Comment (exu) [reply .github/workflows/helm-deploy-testkube-charts-stage.yaml:62]: `integration.testkube.io`? `staging.testkube.io` ?

**Review by exu — CHANGES_REQUESTED** [review]
additional work needed for values file.

File: .github/workflows/helm-deploy-testkube-charts-stage.yaml (hunk 1 above)
Comment (ypoplavs) [reply .github/workflows/helm-deploy-testkube-charts-stage.yaml:62]: As per our discussion, a new issue will be created for Ingress/DNS setup.

**Pull request merged**